.github/workflows/ci.yml

```diff
@@ -112,16 +129,22 @@ jobs:
     # This prevents duplicate CI runs for our own pull requests, whilst preserving the ability to
     # run the CI for each branch push to a fork, and for each pull request originating from a fork.
     if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
+    needs: setup-os-matrix
+    strategy:
+      matrix:
+        library: [ILGPU, ILGPU.Algorithms]
+        framework: [net47, netcoreapp3.1, net5.0]
+      fail-fast: false
```

Comment:
You shouldn't add anything here - we only want one unique job running on Windows that will by default package both libraries for all target frameworks, exactly like today.